crates/iota-json-rpc/src/lib.rs

```diff
@@ -70,8 +70,8 @@ pub struct JsonRpcServerBuilder {
 pub fn iota_rpc_doc(version: &str) -> Project {
     Project::new(
         version,
-        "Iota JSON-RPC",
-        "Iota JSON-RPC API for interaction with Iota Full node. Make RPC calls using https://fullnode.NETWORK.iota.io:443, where NETWORK is the network you want to use (testnet, devnet, mainnet). By default, local networks use port 9000.",
+        "IOTA JSON-RPC",
+        "IOTA JSON-RPC API for interaction with Iota Full node. Make RPC calls using https://api.NETWORK.iota.cafe, where NETWORK is the network you want to use (testnet, devnet, mainnet). By default, local networks use port 9000.",
```

Comment:
No port needed?

Reply:
At least there is a little inconsistency, in the iota-light-client README we mention the :443